deployment/migrations/versions/0033_1c06d0ade60c_calculate_costs_statically.py

```diff
@@ -112,12 +70,15 @@ def do_calculate_costs() -> None:
 
     logger.debug("INIT: CALCULATE COSTS FOR: %r", msg_item_hashes)
 
+    # Build the initial pricing model from DEFAULT_PRICE_AGGREGATE
+    initial_pricing_model = build_default_pricing_model()
```

Comment:
Change a migration already executed it's not a good thing, as it can create differences with the nodes that have the previous migration.